src/store/rootSaga.js

```diff
@@ -1,10 +1,11 @@
-import { all } from 'redux-saga/effects';
-import { profileSaga } from './profile';
-
+import { all, fork } from 'redux-saga/effects';
+import { watchFetchUserData } from './profile';
+import { watchAddAlert } from './app';
 // notice how we now only export the rootSaga
 // single entry point to start all Sagas at once
 export default function* rootSaga() {
   yield all([
-    profileSaga(),
+    fork(watchFetchUserData),
+    fork(watchAddAlert),
```

Comment:
Same, the add alert should be a simple redux function without any async functionality.

Reply:
under `Same` you refer to
> I think the add alert should be a simple redux action. The sagas are for async actions and the watchers belongs to sagas as well.
right?

Reply:
Exactly :)